apps/analytics/views.py

```diff
@@ -89,13 +89,13 @@ def get_submission_count(request, challenge_pk, duration):
     q_params = {'challenge_phase__id__in': challenge_phase_ids}
     since_date = None
     if duration.lower() == 'daily':
-        since_date = timezone.now().date()
+        since_date = timezone.now().replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
@Ayukha Can you add a reason why you need to make this change?

Reply:
Submitted_at is a datetime field so it was throwing warnings on receiving only a date. I used this code to pass the datetime of midnight rather than just today's date in since_date.